Backpack/src/main/java/net/skyscanner/backpack/calendar/BpkCalendar.kt

```diff
@@ -17,11 +23,35 @@ open class BpkCalendar @JvmOverloads constructor(
   defStyle: Int = 0
 ) : ConstraintLayout(context, attrs, defStyle), OnYearChangedListener {
 
+  private var calendarController: BpkCalendarController? = null
+
+  var isSelectionTypeSwitcherVisible: Boolean = false
```

Comment:
I don't think this should live here, this code is only used for testing purposes so we should move it to the calendar stories inside the example app.